catroidTest/src/org/catrobat/catroid/test/utils/StatusBarNotificationManagerTest.java

```diff
@@ -81,6 +83,18 @@ public void testShowOrUpdateNotification() {
 		}
 	}
 
+	public void testUploadRejectedNotification(){
```

Comment:
@obusher: This checkstyle rule and similar ones are waiting to be introduced :wink:. https://github.com/Catrobat/Catroid/pull/1242#discussion_r30169240